src/Controller/TaskController.php

```diff
@@ -48,23 +52,34 @@ public function update(
     #[Route('tasks', name: 'get_tasks', methods: 'GET')]
     public function list(TaskCalendarRepository $taskCalendarRepository, Request $request): JsonResponse
     {
-        $page = (int) $request->query->get('page', 1);
-        $per_page = (int) $request->query->get('per_page', 10);
-        $sort = $request->query->get('sort', null);
+        try {
+            $page = (int) $request->query->get('page', 1);
+            $per_page = (int) $request->query->get('per_page', 10);
+            $sort = $request->query->get('sort', null);
+            $filter = $request->query->get('filter', null);
```

Comment:
W sumie to parametry z `query` możesz chyba przechwycić jako parametr akcji kontrolera. Sprawdź proszę w dokumentacji czy potrzebna będzie do tego dodatkowa konfiguracja czy powinno to działać od razu :)

Reply:
chyba to jest dostępne też od Symfony 6.3
https://symfony.com/blog/new-in-symfony-6-3-query-parameters-mapper

na teraz można to zamknąć z obiekcie i przekazać do metody z repositorium